### PR TITLE
Cleanup unused text

### PR DESCRIPTION
**Pull request by monkey-programmer**
Remove unused text

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -1,2 +1 @@
-console.log('Hello World from Rikky');
-
+console.log('Hello World from, Rikky!');
```
Comment (vietlinh2001) [inline index.js]: Comma is not neccessary

**Review by vietlinh2001 — CHANGES_REQUESTED** [review]

File: index.js (hunk 1 above)
Comment (monkey-programmer) [reply index.js]: OK, got it! 👍

**Comment by monkey-programmer** [comment]
@vietlinh2001 I just removed the comma, please check again

**Comment by vietlinh2001** [comment]
OK, thanks :100:

**Review by vietlinh2001 — APPROVED** [review]

**Pull request merged**